'use strict';

// если мы рассмотрим try..catch внутри функции, то станет ясно, что если в них есть return, то последующая очистка не сработает, т.к. мы выйдем из функции.

// finally же гарантирует очистку
